files/fr/web/performance/how_long_is_too_long/index.html

```diff
@@ -0,0 +1,28 @@
+
+<p>En optimisant les performances de votre site, visez un temps de premier chargement du contenu ambitieux, comme par exemple 5 secondes sur un réseau mobile 3G, et 1,5 secondes sur un réseau Wifi, avec peut-être des objectifs de chargement de page encore plus ambitieux pour les chargements de page ultérieurs, en tirant parti des <i lang="en">service workers</i> et de la mise en cache. Il y a plusieurs moments différents pour initialiser le chargement de la page, le chargement des ressources additionnelles, la réponse à l'interaction de l'utilisateur ou de l'utilisatrice, et pour proposer des animations fluides :</p>
+
+<h3 id="idling_goal">Objectifs concernant le statut (en anglais <i lang="en">idling</i>)</h3>
+
+<p>Les navigateurs s'occupe des tâches les unes après les autres (même si des tâches de fond peuvent être prises en charge par les <i lang="en">web workers</i>). Cela signifie que l'interaction avec le visiteur, la peinture de la page et l'exécution des scripts ont lieu lors d'un processus unique. Si le processus est occupé à exécuter du code JavaScript complexe, le processus principal ne sera pas disponible pour réagir à la saisie du visiteur (par exemple le fait d'appuyer sur un bouton). Pour cette raison, le périmètre de l'exécution des scripts devrait être limité et divisé en plusieurs bouts de code pouvant chacun être exécuté en 50 millisecondes ou moins. Cela rend le processus disponible pour l'interaction avec la personne visitant le site.</p>
+
+<h3 id="animation_goal">Objectifs concernant les animations</h3>
+
+<p>Pour les animations de défilement et autres animations qui doivent être fluides et <i lang="en">responsive</i>, le contenu est repeint à 60 images par seconde (60 fps), ce qui fait une fois toutes les 16,7&nbsp;millisecondes. Ces 16.7&nbsp;millisecondes incluent la gestion des scripts, le <i lang="en">reflow</i> et le <i lang="en">repaint</i>. Un document met environ 6&nbsp;millisecondes pour rendre une image, ce qui laisse environ 10&nbsp;millisecondes au reste. Tout ce qui prend moins de 60 images par secondes risque d'apparaître cassé, surtout s'il s'agit de changements d'images.</p>
+
+<h3 id="responsiveness_goal">Objectifs concernant l'adaptativité</h3>
```

Comment:
I think "responsiveness" here is better translated by "réactivité" as it does not revolves around form factors and the like of RWD.

Reply:
Fixed in https://github.com/mdn/translated-content/commit/8a5dbb598d0e4d7f8ab9cb68ca6db216ea9eafb8